src/sonic-yang-models/tests/yang_model_tests/tests/mclag.json

```diff
@@ -0,0 +1,36 @@
+{
+    "MCLAG_VALID_DOMAIN_CFG_TEST": {
+	"desc": "valid mclag configuration test"
+    },
+    "MCLAG_VALID_DOMAIN_PEERLINK_ETH_TEST": {
+	"desc": "valid mclag configuration test with peer link ethernet"
+    },
+    "MCLAG_INVALID_DOMAIN_IDS_TEST": {
+	"desc": "configure mclag domain id in range"
+    },
+    "MCLAG_INVALID_KEEPALIVE_TEST": {
+	"desc": "configure mclag keepalive values in range"
+    },
+    "MCLAG_INVALID_SESSION_TIMEOUT_TEST": {
+	"desc": "configure mclag session timeout in range"
+    },
+    "MCLAG_INVALID_KEEPALIVE_MUST_COND_TEST": {
+	"desc": "Invalid session and keepalive multiplier",
+	"eStrKey" : "Must",
```

Comment:
If you have a error tag or message in Yang file, then libyang will throw different error message. In that case, current eStrKey as "must" is not correct.

Reply:
done